karma-story-editor.config.cjs

```diff
@@ -68,6 +68,14 @@ module.exports = function (config) {
         served: true,
         nocache: false,
       },
+      {
+        pattern:
+          'https://fonts.googleapis.com/css2?display=auto&family=Roboto%3Aital%2Cwght%400%2C100%3B0%2C300%3B0%2C400%3B0%2C500%3B0%2C700%3B0%2C900%3B1%2C100%3B1%2C300%3B1%2C400%3B1%2C500%3B1%2C700%3B1%2C900',
+        type: 'css',
+        included: true,
+        served: true,
+        nocache: false,
+      },
```

Comment:
Interesting 👀 

Could you elaborate on this?

Reply:
absolutely, if this ends up sticking i'll add to the description. For now: 

I did this based on searching for other people having issues with running karma when adding in external fonts as `<Link />` - I know that we've been using Google fonts for a while but the way we're doing it is new and how they are getting sent out for has been updated. 
Adding this in fixed this error: `"Error loading remote stylesheet SecurityError: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules"` which pops up a lot (here's one as an example: https://github.com/google/web-stories-wp/runs/4704301224?check_suite_focus=true#step:6:498)

Reply:
looks like it needs to be a specific string. https://karma-runner.github.io/6.3/config/files.html#loading-files-from-another-server
> Globing is not support, so each URL must be specified as a separate pattern.

Reply:
Hmm but the first sentence there says "Patterns _can_ be an absolute URL".

I understand it as:

* Use a string with an absolute URL. No glob supported.
* Use a `pattern`, glob supported

All the examples there indicate the same to me there.

Could you perhaps give it a try if you haven't already?

Reply:
i did try but i'll do it again and post the logs to double check

Reply:
hi hi, yup - forgot to escape the backslashes and also prettier was sneaking in what it thinks is better. we're good on this front now. thanks for pushing it.